### PR TITLE
InfluxDB Clustered documentation

### PR DESCRIPTION
**Pull request by sanderson**
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud-dedicated/admin/databases/update.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ related:
 ---
 
 Use the [`influxctl database update` command](/influxdb/cloud-dedicated/reference/cli/influxctl/database/update/)
-to update a database in your {{< cloud-name >}} cluster.
+to update a database in your {{< product-name omit=" Clustered" >}}.
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/admin/databases/update.md:22]: Missing a word here? I'm not sure what the output is supposed to be.

File: content/influxdb/cloud-dedicated/admin/tokens/update.md [hunk 2]
```diff
@@ -15,11 +15,12 @@ list_code_example: |
...
     <TOKEN_ID>
   ```
-alt_engine: /influxdb/cloud-serverless/admin/tokens/update-tokens/
+alt_links:
+  serverless: /influxdb/cloud-serverless/admin/tokens/update-tokens/
 ---
 
 Use the [`influxctl token update` command](/influxdb/cloud-dedicated/reference/cli/influxctl/token/update/)
-to update a database token's permissions in your {{< cloud-name >}} cluster.
+to update a database token's permissions in your {{< product-name omit=" Clustered" >}}.
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/admin/tokens/update.md:23]: Missing a word?

File: content/influxdb/cloud-dedicated/guides/api-compatibility/v1/_index.md [hunk 3]
```diff
@@ -337,20 +337,20 @@ curl -i 'https://cluster-id.influxdb.io/write?db=DATABASE_NAME&precision=s' \
...
 
 ##### v1 CLI (not supported)
 
-Don't use the v1 CLI with {{% cloud-name %}}.
+Don't use the v1 CLI with {{% product-name %}}.
 While it may coincidentally work, it isn't officially supported.
 
 #### Client libraries
 
 Use language-specific [v1 client libraries](/influxdb/v1.8/tools/api_client_libraries/) and your custom code to write data to InfluxDB.
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/guides/api-compatibility/v1/_index.md:350]: TODO: point link to product/reference/client-libraries

File: content/influxdb/cloud-serverless/admin/billing/pricing-plans.md [hunk 4]
```diff
@@ -11,7 +11,8 @@ menu:
   influxdb_cloud_serverless:
     parent: Manage billing
     name: Pricing plans
-alt_engine: /influxdb/cloud/account-management/pricing-plans/
+alt_links:
+  cloud: /influxdb/cloud/account-management/pricing-plans/
 ---
 
 InfluxDB Cloud Serverless offers a [Free Plan](#free-plan), a [Usage-Based Plan](#usage-based-plan) to pay as you go, and a discounted [Annual Plan](#annual-plan).
```
Comment (jstirnaman) [inline content/influxdb/cloud-serverless/admin/billing/pricing-plans.md:18]: TODO: replace with product name shortcode

File: content/influxdb/cloud-serverless/guides/api-compatibility/v1/_index.md [hunk 5]
```diff
@@ -324,21 +324,21 @@ curl -i 'https://cloud2.influxdata.com/write?db=BUCKET_NAME&rp=RETENTION_POLICY&
...
 
 ##### v1 CLI (not supported)
 
-Don't use the v1 CLI with {{% cloud-name %}}.
+Don't use the v1 CLI with {{% product-name %}}.
 While it may coincidentally work, it isn't officially supported.
 
 #### Client libraries
 
 Use language-specific [v1 client libraries](/influxdb/v1.8/tools/api_client_libraries/) and your custom code to write data to InfluxDB.
```
Comment (jstirnaman) [inline content/influxdb/cloud-serverless/guides/api-compatibility/v1/_index.md:338]: TODO: point to product/reference/client-libraries

**Review by jstirnaman — APPROVED** [review]
A few small corrections I can do tomorrow. Thanks!

File: content/influxdb/cloud-dedicated/admin/databases/update.md (hunk 1 above)
Comment (sanderson) [inline content/influxdb/cloud-dedicated/admin/databases/update.md:22]:
```suggestion
to update a database in your {{< product-name omit=" Clustered" >}} cluster.
```

File: content/influxdb/cloud-dedicated/admin/tokens/update.md (hunk 2 above)
Comment (sanderson) [inline content/influxdb/cloud-dedicated/admin/tokens/update.md:23]:
```suggestion
to update a database token's permissions in your {{< product-name omit=" Clustered" >}} cluster.
```

File: content/influxdb/cloud-dedicated/get-started/write.md [hunk 6]
```diff
@@ -1039,8 +1039,8 @@ _The tutorial assumes using Maven version 3.9 and Java version >= 15._
     2.  Calls `InfluxDBClient.getInstance()` to instantiate a client configured
         with InfluxDB credentials.
 
-        - **`host`**: your {{% cloud-name %}} cluster URL
-        - **`database`**: the name of the {{% cloud-name %}} database to write to
+        - **`host`**: your {{% product-name omit=" Clustered" %}} cluster URL
+        - **`database`**: the name of the {{% product-name %}} database to write to
         - **`token`**: an [database token](/influxdb/cloud-dedicated/admin/tokens/) with _write_ access to the specified database.
```
Comment (sanderson) [inline content/influxdb/cloud-dedicated/get-started/write.md:1044]:
```suggestion
        - **`token`**: a [database token](/influxdb/cloud-dedicated/admin/tokens/) with _write_ access to the specified database.
```

**Pull request merged**